app/content/views/event.py

```diff
@@ -43,23 +43,22 @@ class EventViewSet(BaseViewSet, ActionMixin):
     search_fields = ["title"]
 
     def get_queryset(self):
-        """
-            Return all non-expired events by default.
-            Filter expired events based on url query parameter.
-        """
-
-        if "expired" in self.request.query_params:
-            expired = (
-                self.request.query_params.get("expired", "false").lower() == "true"
-            )
-            midday_yesterday = midday(yesterday())
-            midday_today = midday(now())
-            time = midday_today if midday_today < now() else midday_yesterday
-            if expired:
-                return self.queryset.filter(end_date__lt=time).order_by("-start_date")
-            return self.queryset.filter(end_date__gte=time)
+        """Return all non-expired events by default if not filtering on start or end-date"""
+
+        midday_yesterday = midday(yesterday())
+        midday_today = midday(now())
+        time = midday_today if midday_today < now() else midday_yesterday
+
+        if (
+            "end_range" in self.request.query_params
+            or "start_range" in self.request.query_params
+        ):
+            return self.queryset
```

Comment:
Er dette nødvendig?

Reply:
Ja. Problemet her er at vi har flere felter som filtrerer på det samme, pluss at vi skal ha default filtrering i tillegg. Hvis man har default filtrering på expired=False, så vil det ikke være mulig å hente ut tidligere arrangementer med `end_range` og `start_range` uten sjekken over.

Reply:
Kunne fjernet hele `get_queryset` hvis vi droppet `expired` og heller bare sender inn `start_range=nå` fra frontend, men det krever i såfall en del refaktorering

Reply:
@Her0elt har du en annen løsning evt?

Reply:
hvis det funker nå så tenker jeg, kjør på nå så får vi ta en titt for å forbedre logikken